chia/cmds/data.py

```diff
@@ -301,8 +320,8 @@ def add_mirror(id: str, amount: int, urls: List[str], fee: Optional[str], data_r
     )
 
 
-@data_cmd.command("delete_mirror", short_help="")
-@click.option("-i", "--id", help="", type=str, required=True)
+@data_cmd.command("delete_mirror", short_help="Delete an owned mirror by its coin id")
+@click.option("-i", "--id", help="Store id", type=str, required=True)
```

Comment:
Isn't this the mirror coin id?

```suggestion
@click.option("-i", "--id", help="Coin id", type=str, required=True)
```